### PR TITLE
#21252 adding the ability to the velocity actionlet script to abort s…

### LINKED ISSUE #21252: Add the ability to stop/abort a workflow on velocity script actionlet
Currently if an user calls

```
$workflow.abortProcessor()
```

The workflow is aborted by not stopped (which means next actionlets after the abort will continue the execution)

The idea is, if an user calls on a velocity script actionlet the abort, the workflow is aborted and stopped.

### PR DESCRIPTION
**Pull request by jdotcms**
add to the velocity actionlet script the ability to stop and abort the workflow

### CODE REVIEW

**Review by erickgonzalez — APPROVED** [review]

File: dotCMS/src/main/java/com/dotmarketing/portlets/workflows/actionlet/VelocityScriptActionlet.java [hunk 1]
```diff
@@ -117,4 +119,9 @@ public void executeAction(final WorkflowProcessor processor,
             throw new WorkflowActionFailureException(e.getMessage(), e);
         }
     }
+
+    @Override
+    public boolean stopProcessing() {
```
Comment (freddyucv) [inline dotCMS/src/main/java/com/dotmarketing/portlets/workflows/actionlet/VelocityScriptActionlet.java:124]: doc?

File: dotCMS/src/main/java/com/dotmarketing/portlets/workflows/actionlet/VelocityScriptActionlet.java [hunk 2]
```diff
@@ -108,6 +109,7 @@ public void executeAction(final WorkflowProcessor processor,
                             "contentlet", processor.getContentlet(),
                             "content", processor.getContentlet()));
 
+            this.stop = processor.abort();
```
Comment (freddyucv) [inline dotCMS/src/main/java/com/dotmarketing/portlets/workflows/actionlet/VelocityScriptActionlet.java:112]: I think the abort method is not really aborting it, right, so why is it not called something like isAborted?

**Review by freddyucv — CHANGES_REQUESTED** [review]

File: dotCMS/src/main/java/com/dotmarketing/portlets/workflows/actionlet/VelocityScriptActionlet.java (hunk 2 above)
Comment (jdotcms) [reply dotCMS/src/main/java/com/dotmarketing/portlets/workflows/actionlet/VelocityScriptActionlet.java:112]: you totally right, but it is legacy old stuff

File: dotCMS/src/main/java/com/dotmarketing/portlets/workflows/actionlet/VelocityScriptActionlet.java (hunk 1 above)
Comment (jdotcms) [reply dotCMS/src/main/java/com/dotmarketing/portlets/workflows/actionlet/VelocityScriptActionlet.java:124]: I am overriding a super class method, it is doc there

**Review by dsilvam — APPROVED** [review]

**Review by nollymar — APPROVED** [review]

**Pull request merged**